src/NServiceBus.AcceptanceTests/BestPractices/When_subscribing_to_command_bestpractices_disabled_on_endpoint.cs

```diff
@@ -7,6 +7,7 @@
 
     public class When_subscribing_to_command_bestpractices_disabled_on_endpoint : NServiceBusAcceptanceTest
     {
+        [Ignore("not supported via code first API")]
```

Comment:
I wanted to check whether this is something we want to fix or not. Either way, this seems to be unrelated to this test. I raise a follow up issue to discuss this.